src/utils/vc.js

```diff
@@ -327,15 +415,16 @@ export function validateCredentialSchema(credential, schema) {
  * as we support more APIs there are more details associated with each API. Only Dock is supported as of now.
```

Comment:
This function assumes `credential` is in expanded form. Please mention this somwhere in the doc comment.